opentracing-shim/src/test/java/io/opentelemetry/opentracingshim/testbed/clientserver/TestClientServerTest.java

```diff
@@ -47,17 +48,33 @@ void after() throws InterruptedException {
   @Test
   void test() throws Exception {
     Client client = new Client(queue, tracer);
-    client.send();
+    client.send(false);
+    verify();
+  }
+
+  @Test
+  public void testUpperCaseKeys() throws Exception {
+    Client client = new Client(queue, tracer);
+    client.send(true);
+    verify();
+  }
 
+  private void verify() {
     await().atMost(Duration.ofSeconds(15)).until(finishedSpansSize(otelTesting), equalTo(2));
 
     List<SpanData> finished = otelTesting.getSpans();
     assertThat(finished).hasSize(2);
 
     finished = sortByStartTime(finished);
     assertThat(finished.get(1).getTraceId()).isEqualTo(finished.get(0).getTraceId());
-    assertThat(finished.get(0).getKind()).isEqualTo(Kind.CLIENT);
-    assertThat(finished.get(1).getKind()).isEqualTo(Kind.SERVER);
+    Kind firstSpanKind = finished.get(0).getKind();
+    if (firstSpanKind == Kind.CLIENT) {
```

Comment:
do we need to `sortByStartTime` any more if we're not going to rely on the order? Also, why did this change make the times change?

Reply:
I removed sorting. I guess that client and server spans are created too fast and we have kind of floating of nano seconds. If I put `sleep(100)` in the client then sorted order is `client->server`, I preder to avoid adding `sleep`.